setup.py

```diff
@@ -20,18 +20,16 @@
 INSTALL_REQUIRES = [
     "transformers==4.36.2",
     "torch@https://download.pytorch.org/whl/cpu/torch-2.0.0%2Bcpu-cp38-cp38-linux_x86_64.whl#sha256=354f281351cddb590990089eced60f866726415f7b287db5105514aa3c5f71ca",
-    "datasets==2.17.0",
-    "fsspec==2023.10.0",
-    "multidict==6.0.5",
-    "urllib3==1.26.6",
-    "sentencepiece==0.2.0",
+    "datasets==2.7.0",
+    "fsspec==2023.6.0",
+    "multidict==6.0.4",
+    "urllib3<2",
```

Comment:
other package changes seem okay. But the urllib3<2 is tested well? it breaks unevenly.

Reply:
Ran this with CI on #92 PR in internal Repo